src/main/java/HWS/HW19/pages/MainPage.java

```diff
@@ -0,0 +1,20 @@
+package HWS.HW19.pages;
+
+import HWS.HW19.componentes.HeaderComponent;
+import HWS.Util.BasePage;
+import org.openqa.selenium.WebDriver;
+
+public class MainPage extends BasePage {
+    HeaderComponent headerComponent;
+    public MainPage(WebDriver driver) {
+        super(driver);
+        this.headerComponent = new HeaderComponent(driver);
+    }
+    public void openMainPage(){
+        driver.get("https://automationexercise.com/");
+    }
+
+    public void openContactUsPage() { // Do I need this method? Or better to use HeaderComponent directly?
+        headerComponent.navigateToContactUsPage();
+    }
```

Comment:
As HeaderComponent is a component of a page but not a separate page, it is better to leave as it is now.